Arelle/Arelle
# Beta command line option to remove lxml from XBRL model hierarchy

#### Reason for change
This is part of a larger effort to decouple the XBRL models from lxml.

#### Description of change
Creates a metaclass and command line option to swap out the object model lxml base classes.

#### Steps to Test
* CI to verify the existing path is not impacted
* Try to open a document from the command line using the `--betaObjectModel` argument.
  * You should get an unhelpful error from the lxml parser that the lookup class doesn't extend an lxml base class `TypeError: result of class lookup must be subclass of <class 'type'>, got <class 'arelle.model.ModelTransition.LXMLModelTransitionMetaclass'>`

**review**:
@Arelle/arelle
